Lib/pickletools.py

```diff
@@ -2484,7 +2482,7 @@ def dis(pickle, out=None, memo=None, indentlevel=4, annotate=0):
                     assert opcode.name == "POP"
                     numtopop = 0
             else:
-                errormsg = markmsg = "no MARK exists on stack"
+                errormsg = "no MARK exists on stack"
```

Comment:
I removed this because:

* this was the only case when the error message was also included in the output
* all other comments in the output are enclosed in parentheses, so this one was outlier